scene/3d/audio_stream_player_3d.cpp

```diff
@@ -847,15 +847,15 @@ void AudioStreamPlayer3D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_stream_playback"), &AudioStreamPlayer3D::get_stream_playback);
 
 	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "stream", PROPERTY_HINT_RESOURCE_TYPE, "AudioStream"), "set_stream", "get_stream");
-	ADD_PROPERTY(PropertyInfo(Variant::INT, "attenuation_model", PROPERTY_HINT_ENUM, "Inverse,Inverse Square,Log,Disabled"), "set_attenuation_model", "get_attenuation_model");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "attenuation_model", PROPERTY_HINT_ENUM, "Linear,Square,Logarithmic,Disabled"), "set_attenuation_model", "get_attenuation_model");
```

Comment:
It's a bit confusing IMO that the human readable names, and the documentation, do not match the names of the actual enum values.

Is it linear or inverse? `1/x` is not linear/affine.

Reply:
It's probably inverse indeed – the class reference confused me.

"True" linear attenuation can be achieved by setting the attenuation model to **Disabled** and increasing **Max Distance** to a value above 0.